### PR TITLE
7.1 search fixes

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-5905 (jdk version), https://issues.liferay.com/browse/LRDOCS-6304 (example cluster IP:host), https://issues.liferay.com/browse/LRDOCS-6312 (reindex in new search admin).

### CODE REVIEW

File: discover/deployment/articles/01-deploying-liferay/10-installing-elasticsearch.markdown [hunk 1]
```diff
@@ -39,11 +39,17 @@ Installing Elasticsearch is pretty easy and takes only six steps:
 [`JAVA_HOME` environment variable](https://docs.oracle.com/cd/E19182-01/820-7851/inst_cli_jdk_javahome_t/).
 
 If you have multiple JDKs installed, make sure Elasticsearch and @product@ are
-using the same version. You can specify this in `[Elasticsearch
-Home]/bin/elasticsearch.in.sh`:
```
Comment (lipusz) [inline discover/deployment/articles/01-deploying-liferay/10-installing-elasticsearch.markdown]: "Oracle Open JDK 1.8.0_201" That's not really valid version. It's either Oracle or OpenJDK. Avoiding a lawsuit... :stuck_out_tongue:  @rbohl

Comment (lipusz) [reply discover/deployment/articles/01-deploying-liferay/10-installing-elasticsearch.markdown]: Or am I wrong here? :thinking:

Comment (rbohl) [reply discover/deployment/articles/01-deploying-liferay/10-installing-elasticsearch.markdown]: :thinking: check out our compat matrix: https://web.liferay.com/documents/14/21598941/Liferay+DXP+7.1+Compatibility+Matrix/9f9c917a-c620-427b-865d-5c4b4a00be85

Comment (lipusz) [reply discover/deployment/articles/01-deploying-liferay/10-installing-elasticsearch.markdown]: :+1:

File: discover/deployment/articles/02-installing-search-engine/00-intro.markdown [hunk 2]
```diff
@@ -36,3 +36,12 @@ If you must use either of those features in your search solution's code, use
 Elasticsearch. If you're using @commerce@, use Elasticsearch.
 Otherwise, feel free to use Elasticsearch or Solr to index your portal content.
 
+Another factor to consider in your search engine selection is JDK version. The
+search engine and @product@ must use the same JDK version and distribution
+(e.g., Oracle Open JDK 1.8.0_201).Consult the 
+[Elasticsearch compatibility matrix](https://www.elastic.co/support/matrix#matrix_jvm),
+[Solr's system requirements](https://lucene.apache.org/solr/guide/7_0/solr-system-requirements.html),
```
Comment (lipusz) [inline discover/deployment/articles/02-installing-search-engine/00-intro.markdown]: We don't need to mention Solr here as we use HTTP when communicating with Solr, no serialization/de-serialiazion happens at JVM level.

**Comment by lipusz** [comment]
Finished reviewing. :+1: Added one additional note above. Thank you, Russ! @rbohl

**Comment by rbohl** [comment]
@lipusz can you see my last commit? 

I do not fully understand what you mean, so you'd better review my text. Also, is there a  resource I can look at to understand this point? I'd like to fill in the knowledge gap.

**Comment by rbohl** [comment]
ahh, yes: https://www.elastic.co/blog/found-interfacing-elasticsearch-picking-client#different-client-types

**Comment by lipusz** [comment]
Which of my comment is not clear, @rbohl ?

**Comment by lipusz** [comment]
The one for the Solr jdk thing?

**Comment by rbohl** [comment]
Yes, the reason that Solr's JDK doesn't need to be matched, but Elasticsearch's does. But I think I understand (enough for now anyway). It's because we're using the Java Transport Client not the HTTP client. From the blog I linked:

> It’s also beneficial to run the same JVM update version on both the clients and the servers due to exception serializing and other potential minor differences between updates.

**Comment by lipusz** [comment]
Yes, that's the reason, sorry for not explaining. I was in a bit rush to leave the office... :see_no_evil:

**Comment by lipusz** [comment]
Ready to send to Rich! :+1:

**Comment by rbohl** [comment]
forwarded here: https://github.com/sez11a/liferay-docs/pull/3922

**Comment by lipusz** [comment]
Basically our Elasticsearch REST client story for 7.2 will bring into the product what the Solr integration has always been doing. :smirk:

**Pull request closed without merging**